.rubocop.yml

```diff
@@ -88,6 +88,10 @@ Style/TrailingBlankLines:
 Style/TrailingWhitespace:
   Enabled: true
 
+# Checks for redundant uses of self.
```

Comment:
This is alphabetically ordered, so it should be in the  79, before `Style/StringLiterals`